### ISSUE TITLE
[BUG] EdXposed framework is not (properly) installed

### ISSUE THREAD

**Issue by 00due** [ElderDrivers/EdXposed#828]
I updated riru, and then edxposed, but now edxposed manager says edxposed is not properly installed. I reinstalled manager, tried older versions, checked if riru is working (it is), installed canary version of edxposed. None of them helped. 

**版本信息/Versions**

Android: 10

Magisk: 21.2

Riru: 23.5

EdXposed: 4683 master

**Comment by MlgmXyysd** [comment]
uninstall EdXposed in Magisk, reboot, then reinstall it

**Comment by 00due** [comment]
Rebooted phone and its working.

**Comment by bobwang3** [comment]
I also have the same problem. Today I upgraded my phone's OS from Android 9 to Android 10, Magisk didn't work, and reinstalled Magisk-21.4(21400).zip in recovery. after Magisk worked, I install Riru and Riru-EdXposed, but unfortunately, no matter what the EdXposed manager shows the red words "EdXposed framework is not (properly) installed"
The log of EdXposed manager shows "/data/user_de/0/org.meowcat.edxposed.manager/log/error.log: open failed: ENOENT (No such file or directory)". directory)"
I have tested that it is not a permission problem of /data/user_de/0/org.meowcat.edxposed.manager/, I still get this error after changing to 777.

Android: 10
Magisk: 21.4
Riru: 23.5
EdXposed: v0.5.2.1_4677-master

I have tried removing Riru-EdXposed from Magisk and reinstalling it, all presenting the same failure.

**Comment by MlgmXyysd** [comment]
What does the installation script show when you install it?

**Comment by bobwang3** [comment]
- Copying zip to temp directory
- Installing Riru_-_EdXposed-v0.5.2.1_4677-master(4677).zip
- Device is system-as-root
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: module.prop
***********************
 Riru - EdXposed 
 by solohsu, MlgmXyysd 
***********************
*******************
 Powered by Magisk 
*******************
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: customize.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: zh_CN.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: zh_CN.sh.s
- EdXposed 版本 v0.5.2.1_4677-master
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: verify.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: customize.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: customize.sh.s
- 已验证 customize.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: util_functions.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: util_functions.sh.s
- 已验证 util_functions.sh
cat: can't open '/data/adb/riru/api_version.new': No such file or directory
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: EdXposed.apk
- Android SDK 版本: 29
- Magisk 版本: 21400
- Riru API 版本: 10
- 设备平台: arm64
- 正在解压模块文件
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: EdXposed.apk.s
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: module.prop
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: module.prop.s
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system.prop
- 已验证 EdXposed.apk
- 已验证 module.prop
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system.prop.s
- 已验证 system.prop
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: sepolicy.rule
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: sepolicy.rule.s
- 已验证 sepolicy.rule
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: post-fs-data.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: post-fs-data.sh.s
- 已验证 post-fs-data.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: uninstall.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: uninstall.sh.s
- 已验证 uninstall.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/framework/eddalvikdx.dex
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/framework/eddalvikdx.dex.s
- 已验证 system/framework/eddalvikdx.dex
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/framework/eddexmaker.dex
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/framework/eddexmaker.dex.s
- 已验证 system/framework/eddexmaker.dex
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/framework/edservice.dex
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/framework/edservice.dex.s
- 已验证 system/framework/edservice.dex
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: sandhook.dex
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: sandhook.dex.s
- 已验证 system/framework/sandhook.dex
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: yahfa.dex
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: yahfa.dex.s
- 已验证 system/framework/yahfa.dex
- 正在解压 arm 运行库
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/lib/libriru_edxp.so
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/lib/libriru_edxp.so.s
- 已验证 system/lib/libriru_edxp.so
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/lib/libsandhook.edxp.so
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/lib/libsandhook.edxp.so.s
- 已验证 system/lib/libsandhook.edxp.so
- 正在解压 arm64 运行库
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/lib64/libriru_edxp.so
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/lib64/libriru_edxp.so.s
- 已验证 system/lib64/libriru_edxp.so
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/lib64/libsandhook.edxp.so
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: system/lib64/libsandhook.edxp.so.s
- 已验证 system/lib64/libsandhook.edxp.so
- 正在预装管理器
  ! 管理器预装失败! 请手动安装 EdXposed Manager
- 正在创建配置目录
  - 使用新目录 EwidRLit3yYku8Gg
- 正在复制框架运行库
rm: can't remove '/data/misc/edxp_EwidRLit3yYku8Gg/cache/*': No such file or directory
- 正在移除老旧配置
- 正在复制文件
- 欢迎使用 EdXposed v0.5.2.1_4677-master!
- Installing custom sepolicy rules
- Done

**Comment by NULL0B** [comment]
I was on v0.5.2.1_4677 YAHFA Android 11 One Ui 3.0 then I upgraded to v0.5.2.2_4683 I got this error. So i uninstalled it reboot and then installed it. That solved the issue as the author says. I enabled Sandhook and im feeling it slower than YAHFA is that right? I also enabled resource Hooks this time. My edxposed always has been enabled globally.

**Comment by mario0318** [comment]
Why was this issue closed? To this day I'm also experiencing this issue, the one about the file permission: "/data/user_de/0/org.meowcat.edxposed.manager/log/error.log: open failed: ENOENT (No such file or directory)". directory)"

It's the one thing that has made me switch over to LSPosed.

Not to mention EdXposed for whatever reason doesn't seem to read the modules scope list that comes with the new modules yet LSPosed does? What's going on with these recent releases?

**Comment by MlgmXyysd** [comment]
@mario0318 That's your own problem.
EdXposed supports scope list reading as early as the previous update.

**Comment by omgfrost** [comment]
Hello,
Im on custom rom android 10 and today i found first same top post screenshot with framework not installed,so i uninstalled with magisk,rebooted,installed from latest alpha in app always via magisk,rebooted and nt solved..the red not installed alert is there..
What i should do?
Please, help
![Screenshot_20210414-161346948](https://user-images.githubusercontent.com/14861120/114725018-71eeb500-9d3c-11eb-84dd-c8c2539a5227.jpg)

**Comment by omgfrost** [comment]
Bump please, i need my modules back!

**Comment by omgfrost** [comment]
I DID IT!
![Screenshot_20210419-210042686](https://user-images.githubusercontent.com/14861120/115289054-76f3a000-a152-11eb-9328-05267570f177.jpg)
All i did this time is uninstall riruEdxposed from magisk,reboot with button,go in magisk again,search for it, reinstalled and reboot.
BUT THIS TIME I WAITED MORE AND PHONE HAD TO MAKE 2 RESTARTS FOR IT TO BE PROPERLY INSTALLED!
ALL YOU HAVE TO DO IS JUST WAIT!
Hope this helps

**Comment by omgfrost** [comment]
BUMP PLEASE,
I  just checked again after above post today and during this time i might have rebooted the phone and again this
![Screenshot_20210426-171535472](https://user-images.githubusercontent.com/14861120/116107620-79ef1300-a6b3-11eb-88b3-21d130a49664.jpg)
This time i tried everything...
Uninstalled riru, uninstalled xposed either riruxposed and the one from exposed manager either alpha and canary...
Rebooted phone at every step....
Nothing worked this time
So please focus on this now...
I really need my modules back for working purpose!
Log:
![Screenshot_20210426-172622463](https://user-images.githubusercontent.com/14861120/116108860-95a6e900-a6b4-11eb-92f6-1315ee56512d.jpg)

**Comment by Catwoolfii** [comment]
In my case, the problem was that the app was installed on the SD-card! After the application was moved to the internal memory, this problem disappeared.

**Comment by omgfrost** [comment]
> BUMP PLEASE,
> I just checked again after above post today and during this time i might have rebooted the phone and again this
> ![Screenshot_20210426-171535472](https://user-images.githubusercontent.com/14861120/116107620-79ef1300-a6b3-11eb-88b3-21d130a49664.jpg)
> This time i tried everything...
> Uninstalled riru, uninstalled xposed either riruxposed and the one from exposed manager either alpha and canary...
> Rebooted phone at every step....
> Nothing worked this time
> So please focus on this now...
> I really need my modules back for working purpose!
> Log:
> ![Screenshot_20210426-172622463](https://user-images.githubusercontent.com/14861120/116108860-95a6e900-a6b4-11eb-92f6-1315ee56512d.jpg)

IM SORRY!
Ive posted in the wrong section but i was via phone and i didnt notice there were Edxposed Framework and Manager separate.
So here is the link to the solution:
[My Framework Solution](https://github.com/ElderDrivers/EdXposedManager/issues/185#issuecomment-834526311)

**Comment by Cjankhavaniael** [comment]
This is very Easy to solve. I won't say it now, because maybe no one is needing the answer. Just ping me if someone needs.

**Comment by mario0318** [comment]
> This is very Easy to solve. I won't say it now, because maybe no one is needing the answer. Just ping me if someone needs.

What an absolutely silly and selfish thing to do. You know very well there are folks looking for this solution. Why make it a secret for future users to find this and NOT be able to apply your solution themselves? So unless you're referring to actual program source code solution that you don't wish to reveal to users but to the developers, then simply post your solution! It hurts no one to do so.

**Comment by Nek0kawa1** [comment]
> 
> 
> > This is very Easy to solve. I won't say it now, because maybe no one is needing the answer. Just ping me if someone needs.
> 
> What an absolutely silly and selfish thing to do. You know very well there are folks looking for this solution. Why make it a secret for future users to find this and NOT be able to apply your solution themselves? So unless you're referring to actual program source code solution that you don't wish to reveal to users but to the developers, then simply post your solution! It hurts no one to do so.

if you post it, your account will be banned by ElderDrivers. that's why.

**Comment by Cjankhavaniael** [comment]
ok2x for Humanities sake! I wont tell you the exact source code, but for your bonus I will be telling you

> > This is very Easy to solve. I won't say it now, because maybe no one is needing the answer. Just ping me if someone needs.
> 
> What an absolutely silly and selfish thing to do. You know very well there are folks looking for this solution. Why make it a secret for future users to find this and NOT be able to apply your solution themselves? So unless you're referring to actual program source code solution that you don't wish to reveal to users but to the developers, then simply post your solution! It hurts no one to do so.

ok2x for Humanities sake! I wont tell you the exact source code, but for your bonus I will be telling you the simple workaround or fix: here it is 

First of all uninstall everything that has something to do with your EdXpose except the Magisk itself, well OFF COURSE! hahaha :D then, 

Download, flash, and install these: 

magisk-riru-core-v19.7.zip:
https://mega.nz/file/Wj5GGQoQ#zAa8JdXTXhat9MCwTqUKnvbEMBjTHP29PP3YkELfI3Q

EdXposedManager-4.5.4-4540.apk:
https://mega.nz/file/qyxwCKSQ#4myooZuXKRfWdlCaipOW8vu_jFFAZTGnjWovEFJf724

EdXposed-SandHook-v0.4.6.1.4510.zip:
https://mega.nz/file/iihWlAAJ#-Qeb9fmwxIRfu4Nu-L7nPCAYNDhVWHPC4T8pF4Adsts

HAPPY NOW??? hahahaha

BASICALLY, what im doing was to let a brand new or fresh start of all those things you needed to have a fully functioning EdXposed App in your android. thats it! the updated and proper fix??? Nah! Ill let my team decide what to do with it!

**Comment by Cjankhavaniael** [comment]
> ok2x for Humanities sake! I wont tell you the exact source code, but for your bonus I will be telling you
> 
> > > This is very Easy to solve. I won't say it now, because maybe no one is needing the answer. Just ping me if someone needs.
> > 
> > 
> > What an absolutely silly and selfish thing to do. You know very well there are folks looking for this solution. Why make it a secret for future users to find this and NOT be able to apply your solution themselves? So unless you're referring to actual program source code solution that you don't wish to reveal to users but to the developers, then simply post your solution! It hurts no one to do so.
> 
> ok2x for Humanities sake! I wont tell you the exact source code, but for your bonus I will be telling you the simple workaround or fix: here it is
> 
> Download these:
> 
> magisk-riru-core-v19.7.zip:
> https://mega.nz/file/Wj5GGQoQ#zAa8JdXTXhat9MCwTqUKnvbEMBjTHP29PP3YkELfI3Q
> 
> EdXposedManager-4.5.4-4540.apk:
> https://mega.nz/file/qyxwCKSQ#4myooZuXKRfWdlCaipOW8vu_jFFAZTGnjWovEFJf724
> 
> EdXposed-SandHook-v0.4.6.1.4510.zip:
> https://mega.nz/file/iihWlAAJ#-Qeb9fmwxIRfu4Nu-L7nPCAYNDhVWHPC4T8pF4Adsts
> 
> HAPPY NOW??? hahahaha

BASIC

> ok2x for Humanities sake! I wont tell you the exact source code, but for your bonus I will be telling you
> 
> > > This is very Easy to solve. I won't say it now, because maybe no one is needing the answer. Just ping me if someone needs.
> > 
> > 
> > What an absolutely silly and selfish thing to do. You know very well there are folks looking for this solution. Why make it a secret for future users to find this and NOT be able to apply your solution themselves? So unless you're referring to actual program source code solution that you don't wish to reveal to users but to the developers, then simply post your solution! It hurts no one to do so.
> 
> ok2x for Humanities sake! I wont tell you the exact source code, but for your bonus I will be telling you the simple workaround or fix: here it is
> 
> Download these:
> 
> magisk-riru-core-v19.7.zip:
> https://mega.nz/file/Wj5GGQoQ#zAa8JdXTXhat9MCwTqUKnvbEMBjTHP29PP3YkELfI3Q
> 
> EdXposedManager-4.5.4-4540.apk:
> https://mega.nz/file/qyxwCKSQ#4myooZuXKRfWdlCaipOW8vu_jFFAZTGnjWovEFJf724
> 
> EdXposed-SandHook-v0.4.6.1.4510.zip:
> https://mega.nz/file/iihWlAAJ#-Qeb9fmwxIRfu4Nu-L7nPCAYNDhVWHPC4T8pF4Adsts
> 
> HAPPY NOW??? hahahaha

BASICALLY, what im doing was to let a brand new or fresh start of all those things you needed to have a fully functioning EdXposed App in your android. thats it! the updated and proper fix??? Nah! Ill let my team decide what to do with it!

**Comment by Technologx** [comment]
Hi, I'm running into this issue and I don't know what to do.
![Screenshot_20211218-153215](https://user-images.githubusercontent.com/17693282/146656152-89a2f0bd-3ea5-468a-a653-b26d259c8982.jpg)
![Screenshot_20211218-154005](https://user-images.githubusercontent.com/17693282/146656153-7d822928-e960-42a8-ae0c-19a151be0b5b.jpg)
![Screenshot_20211218-154025](https://user-images.githubusercontent.com/17693282/146656154-fb3d1eaf-48be-43e5-9a9a-91aa23e6e876.jpg)

**Comment by Technologx** [comment]
![Screenshot_20211218-154114](https://user-images.githubusercontent.com/17693282/146656185-374087c9-61a8-49d2-b4af-c5f82ecc7776.jpg)
![Screenshot_20211218-154133](https://user-images.githubusercontent.com/17693282/146656190-a92ef634-1df7-44ca-83b8-0b6f47d190f5.jpg)

**Comment by jfzlma** [comment]
i found a solution.
source: https://t.me/EdXposed/138

Please do NOT update to Riru 26, instead use version 25.4.4.
File is available in above link.

After installation, reboot phone.
![IMAGE 2022-01-05 20:11:16](https://user-images.githubusercontent.com/25582497/148236027-4886723c-c286-4498-a1f3-9230f2f479ee.jpg)
![IMAGE 2022-01-05 20:11:21](https://user-images.githubusercontent.com/25582497/148236037-a6e510a5-1231-439c-83e2-2e4376b8453a.jpg)

**Comment by IzzySoft** [comment]
Perfect timing, @jfzlma – and confirmation that this works (Fairphone 2 here running LineageOS 18.1 / Android 11). Thanks!

**Comment by AcidSoap** [comment]
Hello! I can't install EdXposed on my galaxy S8.
Some info:
OS: LOS 16 (Android 9)
Api v28
Arch: Arm64
Magisk: 24
There is log file of installation
********************************
- Copying zip to temp directory
- Installing EdXposed-v0.5.2.2_4683-master-z-debug.zip
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: module.prop
***********************
 Riru - EdXposed 
 by solohsu, MlgmXyysd 
***********************
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: customize.sh
*******************
 Powered by Magisk 
*******************
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
cat: can't open '/dev/tmp/ru_RU.sh.s': No such file or directory
sha256sum: can't open '/dev/tmp/ru_RU.sh': No such file or directory
- EdXposed version v0.5.2.2_4683-master
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: verify.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: customize.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: customize.sh.s
- Verified customize.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: util_functions.sh
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: util_functions.sh.s
- Verified util_functions.sh
- Android SDK version: 28
- Magisk version: 24300
*********************************************************
! is not installed
! Please install Riru from Magisk Manager
*********************************************************
! Installation failed
****************************************************

**Comment by AcidSoap** [comment]
Also i'm already install Riru v26.1.r524 i don't know why he complain and ask me install them.

**Comment by jfzlma** [comment]
> Also i'm already install Riru v26.1.r524 i don't know why he complain and ask me install them.

@AcidSoap Please do NOT install  Riru 26, instead use version 25.4.4.

**Comment by Sujoy-Barua** [comment]
Solution you can try:::: 

I was having the same issue and I noticed that the magisk modules section was saying riru is disabled because zygisk is enabled. I then went to magisk settings and disabled zygisk. It then fixed the exposed framework issue. It works now!!!!!!!!!!!!!

**Comment by likearoma** [comment]
I solved it !!!

I solved it, my devices are in, android 10(lineage), 9(lineage and samsung original version)
It works !!
be careful ! follow the path I took.

1. magisk install (via TWRP) magistk version 23 apk (and never update)
2. Riru core zip install version 19.5 (never update)  in magisk => system renboot
3. "Riru EdXposed zip" install, 4.2.3. alpha (YAHFA) install in magisk (not sandhook) => system reboot
4. instrall "APK"  "EdXposed installer" version 2.2.5 apk in android OS ( 9 or 10) (never be installed by magisk)

now you can use all apps and module in Xposed framework hahaha

NEVER UPDATE

**Comment by metropical** [comment]
I'm working in aOS9 with the OEM HTC U11.
None of these methods seem to work.
Any ideas?

**Comment by Tromick** [comment]
Since Xposed Framework causes to crash on my ChatGPT application. I needed something else for my S7 Edge Stock Android 8 ROM. The reason I want XPosed is for me able to take screenshots on such as WhatsApp, Chrome, Brave Private Mode.

--------------------

As always nobody can helps my problems on any forums. I found the solution. Here we go:

I felt like something is coming me, that is trouble. LOL I did backup of my ROM. Pulled internal stroage with modification dates.

Since Xposed Framework longed my booting time from a couple of seconds to 15-20 seconds. It drove me crazy. I uninstalled it with "xposed-uninstaller-20160211.zip". Before doing this I knew that I am going to be stuck at boot. No problem... I flashed "system" backup to my device. Of course it booted. 

Since LPosed wants Android 8.1+, I have tried to install EdXPosed. Unfortunatelly, to make it work, I tried so many versions of Riru, EdXPosed Manager, EdXPosed Framework... Accidently I updated my Magisk version from below 23 to 27 and Hiding feature gone. I learned that I could not downgrade Magisk. I was mad asf.

Anyway, I prayed and uninstalled Magisk from the app. Then reboot to recovery and installed Magisk v23, flashed "boot" backup to my device. Of  course it booted again and Magisk v23 WORKED without no problem. So I did downgrade. haha

For two times I handled bootloop, bootstuck issues in first hit. For anybody wondering which version am I using for S7 Edge Stock Android 8 ROM, EdXposed-YAHFA-v0.5.1.4_4655-master-release, riru-v25.4.4-release, Magisk-v23.0, EdXposedManager-4.5.7-45700-org.meowcat.edxposed.manager-release.

I did WhiteList on EdXPosed and now ChatGPT works like fire and I can take screenshots from WhatsApp, Chrome Incognito, Brave Private Mode.

----

I am still good at these things after all those years. Nice!

**Comment by ereinaimer** [comment]
> for S7 Edge Stock Android 8 ROM, EdXposed-YAHFA-v0.5.1.4_4655-master-release, riru-v25.4.4-release, Magisk-v23.0, EdXposedManager-4.5.7-45700-org.meowcat.edxposed.manager-release.


Just tried your combo. It works on my Android 8. Samsung J5 Prime. SM-J570F
Specs:

- Android: 8.0 (STOCK)
- Magisk: v25.2
- Riru: v25.4.4
- EdXposed: v0.5.1.4
- EdXposed Manager: v4.5.7 (45700)

Thanks man @Tromick
